.wordlist.txt

```diff
@@ -0,0 +1,165 @@
+AAI
+accessionId
+addObjectToDrafts
+addObjectToFolder
+Ajv
+api
+apisauce
+asciicast
+asciinema
+Async
+async
+autenticate
+auth
+AUTHDB
+automodule
+autosummary
+backend
+baseURL
+CERTREQS
+CLI
+conf
+const
+createFromJSON
+createFromXML
+createNewDraftFolder
+createSlice
+CSC
+CSCfi
+cscfi
+css
+currentSubmissionType
+DAC
+Dataset
+destructure
+ebi
+EGA
+ega
+ENA
+ena
+enasequence
+ENV
+env
+eppn
+eslint
+eslintrc
+FhJ
+fileName
+folderID
+followingly
+FormData
+formData
+frontend
+FWYs
+genindex
+Github
+github
+githubusercontent
+GroupedBySchema
+gunicorn
+hostname
+href
+html
+http
+https
+INITDB
+inteded
+io
+javascript
+js
+JSON
+json
+JSONContent
+JWK
+Keyfile
+Kubernetes
+li
+localhost
+lt
+makeStyles
+maxdepth
+md
+metadataObjects
+Metdata
+middlewares
+modindex
+mongo
+Mongodb
+mongodb
+mTFEFsWsNUbP
```

Comment:
What is this? :D

Reply:
looks to be something in `validator.rst`

Reply:
Oh yea, it's part of an asciinema video id
```
  <script id="asciicast-FWYs48FhJ1mTFEFsWsNUbP43g" src="https://asciinema.org/a/FWYs48FhJ1mTFEFsWsNUbP43g.js" async></script>
```